src/libexec/zmrestoreblob

```diff
@@ -14,21 +14,80 @@
 # If not, see <https://www.gnu.org/licenses/>.
 # ***** END LICENSE BLOCK *****
 
-#Usage
-# su - zimbra
-# zmrestoreblob -m 3
-# where -m is to specify which mailbox to check. In above case it is mailbox id 3
+=head1 NAME
+zmrestoreblob - restores missing blobs for orphan headers
```

Comment:
1. the POD formatting is a bit off, please put a blank line after each ```=foo``` directive (=head1 for NAME and DESCRIPTION).  See examples in ```$ perldoc perlpod``` for reference.
2. The description seems off to me. What 'headers' are we referring to? Perhaps a description something like the following instead?

> Looks for orphaned messages for an account via ```zmblobcheck``` and restores blobs, if found, in a backup.

Remove mention of 'headers' in the rest of the code/comments.

Reply:
Done suggested changes and pushed, thanks